client/coral-embed-stream/src/containers/Stream.js

```diff
@@ -323,6 +323,7 @@ const mapStateToProps = (state) => ({
   assetUrl: state.stream.assetUrl,
   activeTab: state.embed.activeTab,
   previousTab: state.embed.previousTab,
+  classNames: state.comment.classNames
```

Comment:
I suggest using a name like `commentClassNames` which is more clear on the purpose of this property, to not confuse with classnames on the `Stream` container itself.